apps/mark/backend/schema.sql

```diff
@@ -0,0 +1,22 @@
+create table election (
+  -- enforce singleton table
+  id integer primary key check (id = 1),
+  election_data text not null,
+  precinct_selection text,
+  is_test_mode boolean not null default true,
+  skip_election_hash_check boolean not null default false,
+  polls_state text not null default "polls_closed_initial",
+  ballot_count_when_ballot_bag_last_replaced integer not null default 0,
+  is_sound_muted boolean not null default false,
+  marginal_mark_threshold_override real,
+  definite_mark_threshold_override real,
+  cvrs_backed_up_at datetime,
+  scanner_backed_up_at datetime,
+  created_at timestamp not null default current_timestamp
```

Comment:
I think there are some extra fields copied from VxScan or VxCentralScan that we don't need, e.g `marginal_mark_threshold_override` and `ballot_count_when_ballot_bag_last_replaced`. Might make sense to trim down to just what we're using right now.

Reply:
[267511e](https://github.com/votingworks/vxsuite/pull/3223/commits/267511edc5ea8b3e91a9e8fefa26c0bcd2d3ae9a)